docs/Integrations/payments/revenuecat.md

```diff
@@ -0,0 +1,238 @@
+---
+slug: revenuecat
+title: RevenueCat
+tags: []
+description: Learn how to integrate RevenueCat in your FlutterFlow app.
+sidebar_position: 1
+---
+
+# RevenueCat
```

Comment:
I guess we havent mentioned that RevenueCat doesnt work on web somewhere here? Lets mention that in a warning admonition after the intro and lets call it 
:::warning[Pub.Dev package and Limitations]
----mentioning about the underlying package of RevenueCat and how web is not supported in this package----

Reply:
Beginning of the document because users should know this information before they try implementing this package.

Reply:
Ok, added.